tests/Microsoft.Identity.Test.LabInfrastructure/LabAuthenticationHelper.cs

```diff
@@ -72,9 +70,8 @@ public static async Task<AccessToken> GetLabAccessTokenAsync(string authority, s
             {
                 case LabAccessAuthenticationType.ClientCertificate:
                     var clientIdForCertAuth = String.IsNullOrEmpty(clientId) ? LabAccessConfidentialClientId : clientId;
-                    var certThumbprintForLab = String.IsNullOrEmpty(clientId) ? LabAccessThumbPrint : certThumbprint;
 
-                    cert = CertificateHelper.FindCertificateByThumbprint(certThumbprintForLab);
+                    cert = CertificateHelper.FindCertificateByName(LabAccessCertName);
```

Comment:
Can `TestConstants.AutomationTestCertName` be used here, just so the cert is specified once in our code?

Reply:
Good catch.Done.